src/elm/Types/InterfaceMapping.elm

```diff
@@ -641,3 +642,12 @@ retentionToEnglishString retention =
 
         Stored ->
             "Stored"
+
+databaseRetentionToEnglishString : DatabaseRetention -> String
+databaseRetentionToEnglishString databaseRetention =
```

Comment:
this is already declared in the builder page.
Also, we should avoid defining view code (like how to present data as strings to the users) inside a model.

The code you see just up above was a mistake of mine which i never got the chance to fix

Reply:
I'm sorry but I can't find the declaration you're referring to.
You have a good point about defining View code inside a model; however I would go this way to avoid having related definitions scattered without logic into multiple files. We could move all others companion definitions into the View file, but I'm not this is a subject for this PR.

Reply:
Yea, no point doing this now as this code will be removed in the next development cycle, this was more a reminder to not follow my previous bad habits